packages/sdk/odahuflow/sdk/clients/feedback.py

```diff
@@ -0,0 +1,78 @@
+#
+#    Copyright 2021 EPAM Systems
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+#
+"""
+API client
+"""
+
+from odahuflow.sdk.clients.api import RemoteAPIClient, AsyncRemoteAPIClient
+from odahuflow.sdk.definitions import FEEDBACK_URL
+from odahuflow.sdk.models import FeedbackModelFeedbackResponse, FeedbackModelFeedbackRequest
+
+
+class FeedbackClient(RemoteAPIClient):
+    """
+    HTTP Feedback client
+    """
+
+    def create(
+            self, feedback: FeedbackModelFeedbackRequest, model_name: str, model_version: str, request_id: str
```

Comment:
Generally, the idea to use `FeedbackModelFeedbackRequest` class is correct. But as we discussed it is not generated properly now. `FeedbackModelFeedbackRequest` doesn't allow you to provide any actual payload, it will always be an empty object:
```
>>> from odahuflow.sdk.models import FeedbackModelFeedbackRequest
>>> f = FeedbackModelFeedbackRequest()
>>> f.to_dict()
{}
>>> f.feedback = {"test": 1}
>>> f.to_dict()
{}
```
As well as the `FeedbackModelFeedbackResponse` model. Any response will deserialize into an empty object. I suggest to write these models from scratch now, later we will generate them properly.

Reply:
Agree with keshamin. Let's write these models manually